src/errors/types.js

```diff
@@ -0,0 +1,111 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+
+export const TEST_RUN_ERRORS = {
+    uncaughtErrorOnPage:                               { name: 'uncaughtErrorOnPage', code: 'E1' },
```

Comment:
I doubt if we need to use string codes (`E1`, `E1000`, etc.) for error differentiation. IMHO it's better to just export a regular enum-style object from the package wiht the `embeddingUtils` object ([here](https://github.com/DevExpress/testcafe/blob/v1.0.0/src/embedding-utils.js)) or by some other way:
``js
const TEST_RUN_ERRORS = {
    uncaughtErrorOnPage:  Symbol('uncaughtErrorOnPage')
}

Reply:
I've used human readable error codes.
Such way is used in [DevExteme](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/Errors_and_Warnings/#E1001).

Reply:
Yes, using short error codes can bring some benefits. But maybe we can just use them and get rid of the `name` property in error types?